### PR TITLE
feat(i18n): add portuguese brazil translations

### PR DESCRIPTION
**Pull request by Zuiago**
This adds Portuguese from Brazil translate.

### CODE REVIEW

File: src/assets/i18n/de.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
   "anms.settings.general.language.sk": "Slovakisch",
   "anms.settings.general.language.de": "Deutsch",
   "anms.settings.general.language.fr": "Französisch",
+  "anms.settings.general.language.pt-br": "Portugiesisch",
```
[the remark this answers is not in the document]
Comment (Zuiago) [reply src/assets/i18n/de.json]: Yes i can!

**Comment by tomastrajan** [comment]
@Zuiago thank you very much for your contribution!

File: src/assets/i18n/de.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (timdeschryver) [reply src/assets/i18n/de.json]: Thanks!

**Comment by timdeschryver** [comment]
Seems I didn't send it this morning, but feel free to add yourself as a contributor @Zuiago.
(Via the npm command `contributors:add` )

**Comment by tomastrajan** [comment]
@timdeschryver @Zuiago no worries I can add him ;)

**Comment by tomastrajan** [comment]
Done! But the translation will come live only after the next release, thanks again!

**Pull request merged**